google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/DdlBatch.java

```diff
@@ -121,29 +118,6 @@ public ApiFuture<ResultSet> executeQueryAsync(
       final ParsedStatement statement,
       AnalyzeMode analyzeMode,
       QueryOption... options) {
-    if (options != null) {
```

Comment:
This special handling is no longer needed, as it is generically handled in the connection.